src/Dfe.PlanTech.Web/Controllers/PagesController.cs

```diff
@@ -44,7 +47,7 @@ public IActionResult Error()
     [HttpGet(UrlConstants.ServiceUnavailable, Name = UrlConstants.ServiceUnavailable)]
     public async Task<IActionResult> ServiceUnavailable([FromServices] IConfiguration configuration)
     {
-        var contactLink = await _getEntityFromContentfulQuery.GetEntityById<NavigationLink>(configuration["ContactUs:LinkId"]);
+        var contactLink = await _getEntityFromContentfulQuery.GetEntityById<NavigationLink>(_contactOptions.LinkId);
```

Comment:
What happens if `NotFoundError` throws a `KeyNotFoundException` because it couldn't find this?

Reply:
Its nullable in the view model, so it just won't display that link